actionview/test/template/form_helper/form_with_test.rb

```diff
@@ -1051,6 +1051,20 @@ def test_form_with_label_accesses_object_through_label_tag_builder
     assert_dom_equal expected, @rendered
   end
 
+  def test_form_with_label_namespace
+    form_with(model: Post.new, namespace: "namespace") do |f|
+      concat f.label(:title, for: "post_title")
+      concat f.text_field(:title, id: "post_title")
```

Comment:
What do you think about making these `id`/`for` pairs not what the default would be? I think everything is working as expected, but making these values more unique would help make that more obvious

(and same thing in the other test)

Reply:
I agree, something like `my_title`, similar to what you did in https://github.com/skipkayhil/rails/commit/3fa3714def6691b726f01791113cccc7c65a1f09